codalab/server/bundle_manager.py

```diff
@@ -42,7 +41,7 @@ class BundleManager(object):
     Assigns run bundles to workers and makes make bundles.
     """
 
-    def __init__(self, codalab_manager):
+    def __init__(self, codalab_manager, worker_timeout_seconds):
```

Comment:
Can we make this argument optional? It would make it a bit easier for test setup, so we aren't forced to specify `worker_timeout_seconds`.

Reply:
I set it to the previous default of 60 seconds.